src/components/ProjectDashboard/hooks/useAboutPanel.ts

```diff
@@ -23,14 +23,28 @@ export const useAboutPanel = () => {
     projectMetadata?.telegram,
     projectMetadata?.twitter,
   ])
-  const description = projectMetadata?.description
+  const description = useMemo(
+    () =>
+      projectMetadata?.description
+        ? wrapNonAnchorsInAnchor(projectMetadata?.description)
+        : undefined,
+    [projectMetadata?.description],
+  )
+
   return {
     description,
     socialLinks,
     projectName: projectMetadata?.name,
   }
 }
 
+const wrapNonAnchorsInAnchor = (text: string) => {
+  const urlRegex = /\b((http|https):\/\/?[\w\S(-.:#~!$&'()*+,;=%)]+)/g
```

Comment:
I think this regex is too permissive:
1. The `?` after the second slash means this would match http:/example.com (do we want to do this)
2. The `\S` in the bracket group means this regex will match `'` and `>` which could be dangerous if this text eventually goes into something that interprets the HTML. I think this would allow for arbitrary HTML (and by extension JS) execution. We should explicitly match reserved characters described in RFC 3986, along these lines:

```js
const urlRegex = /\b((http|https)://[a-zA-Z0-9-._~:/?#@\[\]!$&'()*+,;=%]+\b)/g
```

Reply:
I might be off on details, not an expert on this kind of thing.

https://www.rfc-editor.org/rfc/rfc3986

Reply:
Thanks for the suggestion! Have updated. I am less worried about arbitrary code as we are sanitising after this, but still good practice none the less!